OWNERS_ALIASES

```diff
@@ -1,16 +1,16 @@
 aliases:
-  technical-oversight-committee:
-    - evankanderson
-    - grantr
-    - markusthoemmes
-    - mattmoor
-    - tcnghia
-  wg-leads:
-    # eventing
-    - grantr
-    - Harwayne
-    - vaikas
-    - n3wscott
-    - matzew
-    - nachocano
-    - lionelvillard
+  eventing-approvers:
+  - alanfx
+  - aliok
+  - devguyio
+  - jcrossley3
+  - lberk
+  - markusthoemmes
+  - matzew
+  - mgencur
+  - skonto
```

Comment:
In a recent call, we agreed to make this list small and relevant, so maybe we should have only the eventing team here ??

Reply:
I believe the agreement was to reduce the _reviewers_ list so the review/assignment pings first stay within the team.  I'm not sure there's any benefit to reducing the number of people able to actually _approve_ from the greater serverless team?

Reply:
> I honestly think the reviewer pool there needs to be updated

@devguyio that message on slack from today actually made me do this change :-)   

In the past we used the core serverless team, so that other could also approve CI related fixes (due to their knowledge, like Martin)